python-stream/send.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+from rstream import Producer
+
+STREAM_NAME = "hello-python-stream"
+
+
+async def send():
+    async with Producer(
+        host="localhost",
+        username="guest",
+        password="guest",
+    ) as producer:
+        await producer.create_stream(STREAM_NAME, exists_ok=True)
```

Comment:
Can you please add the size retention https://github.com/rabbitmq/rabbitmq-tutorials/pull/383/files#diff-aaa61986b4ca5dbb6e120b1304f70f4014885275501a001306d4879b5389b017R30 ? like 2 GB ? Thank you